lib/Analysis/AffineStructures.cpp

```diff
@@ -2717,51 +2717,6 @@ void FlatAffineConstraints::projectOut(Value *id) {
   FourierMotzkinEliminate(pos);
 }
 
-bool FlatAffineConstraints::isRangeOneToOne(unsigned start,
```

Comment:
Was this just not used anywhere?

Reply:
It just had one user and that's now gone with this update.